src/views/Home.vue

```diff
@@ -1,5 +1,9 @@
 <template>
 	<Content v-shortkey.once="['c']" app-name="mail" @shortkey.native="onNewMessage">
+		<div v-if="isMobile"
+			class="toggle icon-confirm"
+			href="#"
+			@click="$emit('close')" />
```

Comment:
This doesn't work yet, because close is not catched by the parent component. You need to trigger a route navigation as it was done previously in https://github.com/nextcloud/mail/blob/3f1e1302619adee986aa6ab90b752265c696dc02/src/components/MailboxThread.vue#L160-L166

This would then be basically the same approach as in contacts: https://github.com/nextcloud/contacts/commit/c1ec4fce4182cfad4533481464ebe8e7ab766650#diff-d2191123f77aadc793b4343d3561021532924c0b1df3117a4707aecfd0bc012b